topo/sparse/sparse.pyx

```diff
@@ -260,6 +260,16 @@ cdef class csarray_float:
 
         return result
 
+    def toSparseArray(self):
+        """
+        Convert this sparse matrix into scipy.sparse CSR matrix.
+        """
+        cdef numpy.ndarray[int, ndim=1, mode="c"] rowInds
+        cdef numpy.ndarray[int, ndim=1, mode="c"] colInds
+        cdef numpy.ndarray[float, ndim=1, mode="c"] values
+        rowInds, colInds, values = self.getTriplets()
+        return scipy.sparse.csr_matrix((values, (rowInds, colInds)), shape=self.shape)
```

Comment:
Is this the only dependence on SciPy?  Is it possible to use topo.sparse without Scipy?  if it's possible, please move the import of scipy into this function.